manifests/value.pp

```diff
@@ -70,8 +68,9 @@
   # If value_real is an empty string then the default value of the key will be
   # managed.
   registry_value { "${key}\\${value_real}":
+    path => $key,
+    value_name => $value_real,
     type => $type,
     data => $data,
   }
 }
-
```

Comment:
Likely causing the travis failure (the trailing line should be there...IDE editors :facepunch:)